src/items/traits.md

```diff
@@ -71,7 +71,8 @@ Object safe traits can be the base trait of a [trait object]. A trait is
 * `Sized` must not be a [supertrait][supertraits]. In other words, it must not require `Self: Sized`.
 * It must not have any associated constants.
 * It must not have any associated types with generics.
-* All associated functions must either be dispatchable from a trait object or be explicitly non-dispatchable:
+* All associated functions must either be dispatchable from a trait object,
+  or be explicitly or implicitly non-dispatchable.
```

Comment:
that terminology feels weird, `fn foo<T>` is also "implicitly non-dispatchable", isn't it?

All associated functions must either be dispatchable from a trait object or must not be applicable to trait objects
```suggestion
or must not be applicable to trait objects.
```